pages/ksphere/konvoy/latest/reference/cluster-configuration/index.md

```diff
@@ -240,9 +241,10 @@ spec:
 | ----------------------------- | -------------------------------------------------------------------------------------- | ------- |
 | `vpc.ID`                | [AWS VPC][aws_vpc_and_subnets] ID where the cluster should be launched      | `""`    |
 | `vpc.routeTableID`      | [AWS RouteTable][aws_route_table] ID that is used by the subnets in the VPC | `""`    |
+| `vpc.overrideDefaultRouteTable`  | Modify the default route table                                              | `true`  |
```

Comment:
nit: should we update the description `Modify the default route table` to closer match its name? Seems like it was a point of discussion